.github/pull_request_template.md

```diff
@@ -1,27 +1,25 @@
----
-name: Pull Request Template
-about: 'template for PRs that ensures basic information is provided'
-title: '(feat|fix|chore|doc): _description of introduced change_'
----
+<!-- 
+Thanks for your contribution! 
+Please follow the instructions on your PRs title and description.
+aligned title description: '(feat|fix|chore|doc): _description of introduced change_'
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Info: <!- text comments ->  will be hidden from the rendered preview of your PR.
+-->
 
 ## Description
```

Comment:
Why do we need this heading? All content of the PR is a description of the PR.

Reply:
And for this reason we need

```
## Description
```

in the template? I'm not talking about the comment which is fine, but the heading.

Reply:
now i got your point, then your right a separate heading `Description` seems doubeled.

what do you think of some example headings i found?
- [ ] Proposed Changes
- [ ] Describe your changes
- [ ] Detailed Change Description

... that would lead to a clearer or more detailed information, what going to change in the PR

Reply:
For me these headings seems to have the same content with different grade of details in it. Adds this value?

Reply:
FMPOV yes, because some explainations what is going to change with the pr can not summaized less lines.

and for myself i like to separate this in a little section like:

1. done this 
2. view this
3. checkout this
4. please remark your preview checks

how you would like to change it... ? leave it empty, or reduce more on the pr-template?

Reply:
A lot of discussion here :P In my opinion, the current version of the PR does improve the existing template and we should merge it. We can of course continue improving it with additional incremental changes.

Remark to the last comment here: 
A template should guide and not overwhelm. Therefore we should keep it small and simple. Additional information can always be provided by a contributor, but I think too many things to fill out up front might be seen as hassle

Reply:
@carslen is it okay for you to merge it with current changes and have a little chat afterwards to update the pr-template to desired state?